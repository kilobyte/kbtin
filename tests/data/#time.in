#time tmp 345;#showme $tmp
#time tmp 1y 245d 15h 30m 4s;#showme $tmp
#time tmp 11 hours, 59 minutes and 45 seconds;#showme $tmp
#time tmp 2 weeks 1 d;#showme $tmp
#time tmp 0.5h;#showme $tmp
#time tmp 0.25d 0.5m;#showme $tmp
#time tmp 0.5h 10.001;#showme $tmp
