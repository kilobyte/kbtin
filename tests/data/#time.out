345
52759804
43185
1296000
1800
21630
